app/src/main/java/com/readrops/app/fragments/ImageCaptionFragment.java

```diff
@@ -0,0 +1,41 @@
+package com.readrops.app.fragments;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.DialogInterface;
+import android.os.Bundle;
+
+import com.readrops.app.R;
+
+public class ImageCaptionFragment extends DialogFragment {
```

Comment:
Is there a particular reason why you created a dialog fragment instead of using MaterialDialogs lib like the image options one?

Reply:
I don't have much experience with Android apps, so I didn't realise that you were using MaterialDialogs. I will make that change and remove this file.